### DISCUSSION TITLE
getting error while doing git clone

### DISCUSSION THREAD

**Discussion by sreekanthacchu9** [learndevopsonline/learndevopsonline#1690] [Q&A] [answered]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/125342809/f7cc62b4-806e-4f24-8428-d262bc742beb)

**Comment by sreedharm07** [comment]
just try to clone in another folder

**Reply by sreekanthacchu9** [reply, marked as answer]
git it and thanks
